te3/features/dax-scripts.md

```diff
@@ -72,62 +72,65 @@ The syntax for DAX scripts is the following:
 
 ```dax
 <DAX script>:
-MEASURE 'Table name'[Measure name] = <DAX expression>
+MEASURE 'Table name'[Measure name] [= [<DAX expression>]]
     [<Measure properties>]
 
-COLUMN 'Table name'[Column name] = <DAX expression>
+COLUMN 'Table name'[Column name] [= [<DAX expression>]]
     [<Column properties>]
 
-TABLE 'Table name' = <DAX expression>
+TABLE 'Table name' [= [<DAX expression>]]
     [<Table properties>]
 
 CALCULATIONGROUP 'Table name'[Column name]
     [<Calculation Group properties>]
-    CALCULATIONITEM "Item 1" = <DAX expression>
+    CALCULATIONITEM "Item 1" [= [<DAX expression>]]
         [<Calculation Item properties>]
-    CALCULATIONITEM "Item 2" = <DAX expression>
+    CALCULATIONITEM "Item 2" [= [<DAX expression>]]
         [<Calculation Item properties>]
     ...
 
 <Measure properties>:
-    DetailRows = <DAX expression>
-    DisplayFolder = "string"
-    FormatString = "string"
-    Description = "string"
+    DetailRows = [<DAX expression>]
+    DisplayFolder = ["string"]
+    FormatString = ["string" / <DAX expression>]
+    Description = ["string"]
     Visible = TRUE/FALSE
-    KpiStatusExpression = <DAX expression>
-    KpiStatusDescription = "string"
-    KpiStatusGraphic = "string"
-    KpiTrendExpression = <DAX expression>
-    KpiTrendDescription = "string"
-    KpiTrendGraphic = "string"
-    KpiTargetExpression = <DAX expression>
-    KpiTargetDescription = "string"
-    KpiTargetFormatString = "string"
+    KpiStatusExpression = [<DAX expression>]
+    KpiStatusDescription = ["string"]
+    KpiStatusGraphic = ["string"]
+    KpiTrendExpression = [<DAX expression>]
+    KpiTrendDescription = ["string"]
+    KpiTrendGraphic = ["string"]
+    KpiTargetExpression = [<DAX expression>]
+    KpiTargetDescription = ["string"]
+    KpiTargetFormatString = ["string"]
 
 <Column properties>:
-    DisplayFolder = "string"
-    FormatString = "string"
-    Description = "string"
+    DisplayFolder = ["string"]
+    FormatString = ["string"]
+    Description = ["string"]
     Visible = TRUE / FALSE
     Datatype = BOOLEAN / DOUBLE / INTEGER / DATETIME / CURRENCY / STRING
 
 <Table properties>:
-    Description = "string"
+    Description = ["string"]
     Visible = TRUE / FALSE
-    DetailRows = <DAX expression>
+    DetailRows = [<DAX expression>]
 
 <Calculation Group properties>:
-    Description = "string"
+    Description = ["string"]
     Visible = TRUE / FALSE
     Precedence = <integer value>
 
 <Calculation Item properties>
-    Description = "string"
+    Description = ["string"]
     Ordinal = <integer value>
-    FormatString = <DAX expression> 
+    FormatString = [<DAX expression>]
 ```
 
+> [!HINT]
```

Comment:
@otykier I don't believe HINT is a valid syntax here, TIP or NOTE would probably be the right choice

![image](https://github.com/TabularEditor/TabularEditorDocs/assets/11939114/654c36f4-e069-48b3-b21f-4c604f54f2da)

Reply:
You're right, it doesn't render correctly. I'll change it to TIP.

![image](https://github.com/TabularEditor/TabularEditorDocs/assets/8976200/331e7d5b-cc9d-457b-935a-d7e6231474c9)